pynicotine/gtkgui/ui/dialogs/searchfilters.ui

```diff
@@ -17,8 +17,9 @@
         <child>
           <object class="GtkLabel">
             <property name="visible">1</property>
-            <property name="label" translatable="yes">You can use this to refine which results are displayed. The full results
-from the server are always available if you clear all the search terms.
+            <property name="label" translatable="yes">Result filters are used to refine which search results are displayed.
+
+To set the filter, press Enter after typing or clearing your terms. The filter applies immediately to the already received results, and to any more that are returned thereafer. The full results from the server are always available if you clear the filter of all terms. You do not need to perform another search to apply a different filter.
```

Comment:
```suggestion
To set the filter, press Enter after typing or clearing your terms. The filter
applies immediately to the already received results, and to any more that are
returned thereafer. The full results from the server are always available if
you clear the filter of all terms. You do not need to perform another search to
apply a different filter.
```

Reply:
By the way, the section on Size says:
```
k or m to specify kilo or megabytes. &gt;10M will find files larger than 10 megabytes. &lt;4000k will find files smaller than 4000k.
```
Should these be in binary byte types instead? kibi/mebi? If so, it calls into question the shorthand notation of `k` and `m` stated above. Though wrong, perhaps those should be left as-is, rather than requiring typing of `kib`/`KiB` for example, just because it's simpler.

Reply:
Noting that `b` also works for bytes.